gwpy/segments/flag.py

```diff
@@ -1569,6 +1581,25 @@ def populate(self, source=DEFAULT_SEGMENT_SERVER,
                     self[key].active &= segments
         return self
 
+    def copy(self, deep=False):
+        """Build a copy of this dictionary.
+
+        Parameters
+        ----------
+        deep : `bool`, optional, default: `False`
+            perform a deep copy of the original dictionary with a fresh 
```

Comment:
@tjma12, codeclimate is complaining about trailing whitespace on this line, please take a look.